e2e/basic-smoke.spec.js

```diff
@@ -18,4 +18,12 @@ describe('Basic smoke tests', () => {
 		await element(by.id('header-back')).tap()
 		await expect(element(by.id('screen-homescreen'))).toBeVisible()
 	})
+
+	it('should show home screen after tap to exit edit-home screen', async () => {
+		await device.reloadReactNative()
+		await element(by.id('button-open-edit-home')).tap()
+		await expect(element(by.id('screen-homescreen'))).toBeNotVisible()
```

Comment:
You don't need this one in every test; it was just me making sure that the homescreen wasn't actually visible when another screen was active.

Which should really be its own test, and not repeated in every one.